js/utils/inventory.js

```diff
@@ -1,6 +1,7 @@
 import $ from 'jquery';
```

Comment:
This module has gotten a bit more complex than I would like. One of the complications is that it manages both fetching inventory at the store and slug level.

I'll chew on it a bit and if I get some clarity on how to simplify it, I'll probably circle back to it.